pkg/operators/ustack/ustack.go

```diff
@@ -36,7 +36,11 @@ const (
 )
 
 const (
+	// Annotations
 	userStackTargetNameAnnotation = "ebpf.formatter.ustack"
```

Comment:
Not used currently, but I'd suggest to bring it back (and rename it to fit this operator - sth. like `ustack.target`).

Reply:
Since I removed the _raw thing, I will remove this annotation too